docs/sphinx/dev_guide/ci.rst

```diff
@@ -12,168 +12,252 @@
-    pipelines on LC Gitlab resources has restrictions described below. If 
-    you have access to LC resources, you can access additional information about
+  * **GitLab** instances in the Collaboration Zone (CZ) of the Livermore 
+    Computing (LC) Center run builds and tests in LC platform and compiler 
+    environments important to many RAJA user applications. Execution of RAJA CI 
+    pipelines on LC GitLab resources has restrictions, which are described 
+    below. If you have access to LC platforms, you can access additional 
+    information about
     `LC GitLab CI <https://lc.llnl.gov/confluence/display/GITLAB/GitLab+CI>`_
 
-The tools automatically run RAJA builds and tests when a PR is created and 
-when changes are pushed to a PR branch.
+The tools integrate with the RAJA GitHub project and automatically run RAJA 
+builds and tests when a PR is created and when changes are pushed to a PR 
+branch.
 
 The following sections describe basic elements of the operation of the CI tools.
 
 .. _gitlab_ci-label:
 
 =========
-Gitlab CI
+GitLab CI
 =========
 
-The Gitlab CI instance used by the RAJA project lives in the Livermore 
+The GitLab CI instance used by the RAJA project lives in the Livermore 
 Computing (LC) Collaboration Zone (CZ). It runs builds and tests in LC 
-resource and compiler environments important to RAJA user applications at LLNL.
+platform and compiler environments important to RAJA user applications at LLNL.
 
 Constraints
 -----------
 
-Running Gitlab CI on Livermore Computing (LC) resources is constrained by LC 
+Running GitLab CI on Livermore Computing (LC) platforms is constrained by LC 
 security policies. The policies require that all members of a GitHub project 
 be members of the LLNL GitHub organization and have two-factor authentication 
-enabled on their GitHub accounts to automatically mirror a GitHub repo and
-trigger Gitlab CI functionality from GitHub. For compliant LLNL GitHub projects,
-auto-mirroring of the GitHub repo on LC Gitlab is done when changes are pushed 
-to PRs for branches in the RAJA repo, but not for PRs for a branch on a fork of
-the repo. An alternative procedure we use to handle this is described in 
-:ref:`contributing-label`. If you have access to LC resources, you can learn
-more about `LC Gitlab mirroring <https://lc.llnl.gov/confluence/pages/viewpage.action?pageId=662832265>`_.
-
-Gitlab CI (LC) Testing Workflow
+enabled on their GitHub accounts. When these requirements are satisfied, 
+mirroring of a GitHub repo and triggering GitLab CI functionality from GitHub
+can be done. Otherwise, LC GitLab CI checks will not be run for a project. 
+For a compliant LLNL GitHub project, such as RAJA, auto-mirroring of the 
+GitHub repo on LC GitLab is done when changes are pushed to PR branches in the 
+RAJA GitHub project. If you have access to LC platforms, you can learn more 
+about `LC GitLab mirroring <https://lc.llnl.gov/confluence/pages/viewpage.action?pageId=662832265>`_.
```

Comment:
```suggestion
For a compliant LLNL GitHub project, such as RAJA, auto-mirroring of the 
GitHub repo on LC GitLab is done every 30 minutes at best, triggering tests
on new changes in RAJA GitHub project. If you have access to LC platforms,
you can learn more about `LC GitLab mirroring <https://lc.llnl.gov/confluence/pages/viewpage.action?pageId=662832265>`_.
```

Reply:
I am actually investigating the 30 minute rule mentioned here. According to GitLab documentation, a change on GitHub could actually trigger a pipeline on GitLab... if the webhooks are set. And it seems they often are not, but I can’t check by myself on RAJA.

Reply:
We don't have webhooks set, but we do have GitLab and Azure checks required to pass before a PR can be merged. So you're saying that if we set webhooks for GitLab, it may start running sooner after changes are pushed to GitHub? If so, I'll look into that.